README.md

```diff
@@ -72,7 +72,7 @@ const appCredentials = await registerGitHubApp({
     issues: "write",
   },
   // List of events for new installations
-  evedefault_events: ["issues"],
+  default_events: ["issues"],
```

Comment:
Hey @gr2,

This entire property present in the configuration is leading to an error

![image](https://github.com/gr2m/register-github-app/assets/25631971/b1da5f21-5346-4259-83e6-27303d88dff1)

I since discovered that inorder for us to listen for a certain event, then we've gotta also configure the `"hook_attributes"` and that isn't present in the code example..

So, should this be added or should we remove the `default_events` entirely!???

Reply:
good catch! I think we should define `hook_attributes` as well